main_logic/core.py

```
@@ -147,6 +147,9 @@ def __init__(self, sync_message_queue, lanlan_name, lanlan_prompt):
         # 用户活动时间戳：用于主动搭话检测最近是否有用户输入
         self.last_user_activity_time = None  # float timestamp or None
         
+        # 累积本轮完整文本（用于前端显示追踪，实际改写逻辑在 omni_offline_client.py）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)